### PR TITLE
Add[Experiment]: more knowledge base articles under "helpful resources"

### PR DESCRIPTION
**Pull request by teesloane**
[Ticket](https://circleci.atlassian.net/browse/DD-321)

This PR adds selective "helpful resource" links to the sidebar at the right of documents on desktop. The links added are being added as an experiment to track the efficacy of these links.

Screenshot:
Screen Shot 2022-01-27 at 1 13 38 PM

### CODE REVIEW

File: src/js/experiments/kbLinks.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    }
+  });
+}
+
+function trackExperimentalKnowledgeBaseLinks() {
+  $('.experimental-link').click(function () {
+    window.AnalyticsClient.trackAction(
+      'docs-experimental-knowledgebase-link-clicked',
+      {
+        link: $(this).children('a').first().attr('href'),
```
Comment (chantelle11) [inline src/js/experiments/kbLinks.js:20]: Is there a way to make this more rebust then find the first 'a' tag?

**Review by chantelle11 — COMMENTED** [review]
Could you add some screen shots to the PR? It works for me!

File: src/js/experiments/kbLinks.js (hunk 1 above)
Comment (teesloane) [reply src/js/experiments/kbLinks.js]: Fixed!

**Comment by teesloane** [comment]
Updated with a PR and some fixes :)

**Review by chantelle11 — APPROVED** [review]
Great work looks good to me! ✌️👑💕

**Review by Mayuko-Sa — APPROVED** [review]

**Review by rsylvian — APPROVED** [review]
Looks all good to me. Tested on my end. Great job 👍

File: src/js/experiments/kbLinks.js [hunk 2]
```diff
@@ -0,0 +1,31 @@
+function renderExperimentalKnowledgeBaseLinks() {
+  // https://app.optimizely.com/v2/projects/16812830475/experiments/20790151733/variations
```
Comment (tshen815) [inline src/js/experiments/kbLinks.js:2]: would you mind just updating the link? https://app.optimizely.com/v2/projects/16812830475/experiments/21129180862/variations thanks!

**Review by nramabad — APPROVED** [review]
LGTM! Same comment as tony's about updating experiment link

**Review by Mayuko-Sa — APPROVED** [review]

**Review by makotom — APPROVED** [review]

**Pull request merged**